### ISSUE TITLE
Design issue of the Progress page

### ISSUE THREAD

**Issue by georai** [mozilla/voice-web#1053]
@Gregoor here is a screenshot with issues related to progress page:
- Borders for the cards
- Missing share button
- Missing icon for Ready to do 5 more

Link: https://www.dropbox.com/s/jm110ydpz8ob3kw/_qwm-kch.png?raw=1

**Comment by m-branson** [comment]
>Missing icon for Ready to do 5 more

@Gregoor possible to make two new versions of this CTA?
- one would use the mic and route to the speak page
- one would use the play icon and route to the listen page
- Both should be camel case but without the underline
- this will initially be used for those contributors without a profile and then will evolve to be used for all user flows whether logged in/out.
- allowing CTAs with either icon means we can prompt for contribution where we need it most for a given language
- note that as iteration continues eventually the 'edit profile' link will no longer be used in any user flow (as a user menu will be available)

Let me know if there are questions :)

**Comment by Gregoor** [comment]
I think I got it at all, if not lemme know 🙃

**Comment by m-branson** [comment]
Couple more things:
- main content is sitting in the top half of the screen, please vertically center in the viewport (see screenshot)
- when logged out, secondary CTA should be the version with either play or mic icons (this is what I was attempting to outline above, apologies if too wordy and confusing 😞)
- `5/5 Clips` should be `5/5 Clips Submitted`
- the share button doesn't have a [hover state](https://zpl.io/V4Jk7y0)
- I'm seeing a dotted line beneath the content, potentially from the active div? It's visible in the screenshot. It pops up randomly throughout the experience actually. Can we clear that?

Current:
<img width="1438" alt="screen shot 2018-06-06 at 2 08 21 pm" src="https://user-images.githubusercontent.com/18686986/41065514-bbc841a8-6993-11e8-9db1-ed6ff510790d.png">


Desired:
<img width="956" alt="screen shot 2018-06-06 at 2 11 47 pm" src="https://user-images.githubusercontent.com/18686986/41065469-9c46ebfe-6993-11e8-96e2-ef6fd5b4dc57.png">

**Comment by Gregoor** [comment]
From looking at the current version it looks like the cta changes weren't on Staging yet. They are now.

RE centering on taller screens: for people with vertical monitors this might be a bit strange, shall we do it similar to contribution, where we push it a fixed amount further down on taller screens. If so, by how much?

Ty for the feedback, taking care of the rest now.

**Comment by mikehenrty** [comment]
I believe we have handled all this feedback in other bugs. If there are any missing actionable items, let's open a new bug for each. Thanks!
